turbopack/crates/turbopack-browser/src/chunking_context.rs

```diff
@@ -591,6 +591,7 @@ impl ChunkingContext for BrowserChunkingContext {
                 .map(|chunk| self.generate_chunk(**chunk).to_resolved())
                 .try_join()
                 .await?;
+            let other_assets = Vc::cell(assets.clone());
```

Comment:
You wanted to not cell conditionally, but this means that we create an unused cell for every chunk group in prod?